lib/tasks/send_reject_resolution_reminder_notifications.rake

```diff
@@ -0,0 +1,27 @@
+namespace :send_reject_resolution_reminder_notifications do
+  desc 'Send reject resolution reminder notifications'
+  task 'send' => :environment do
+    BATCH_SIZE = 50
+    intakes_to_send_message_to.each do |intake|
+      SendRejectResolutionReminderNotificationJob.perform_later(intake)
+    end
+  end
+
+  def intakes_to_send_message_to
+    ny_intakes = StateFileNyIntake.joins(efile_submissions: :efile_submission_transitions)
+                                  .where(efile_submission_transitions: { to_state: 'notified_of_rejection' })
+                                  .where.not(id: StateFileNyIntake.joins(efile_submissions: :efile_submission_transitions)
+                                                                  .where(efile_submission_transitions: { to_state: 'accepted' })
+                                                                  .select(:id))
+                                  .distinct
+
+    az_intakes = StateFileAzIntake.joins(efile_submissions: :efile_submission_transitions)
+                                  .where(efile_submission_transitions: { to_state: 'notified_of_rejection' })
+                                  .where.not(id: StateFileAzIntake.joins(efile_submissions: :efile_submission_transitions)
+                                                                  .where(efile_submission_transitions: { to_state: 'accepted' })
+                                                                  .select(:id))
+                                  .distinct
+
+    ny_intakes.to_a + az_intakes.to_a
```

Comment:
Instead of iterating over the state intake classes separates, some other new message implementations have been doing both at once using `ApplicationRecord::STATE_INTAKE_CLASS_NAMES.each do |base_class|` so that there aren't multiple blocks to maintain going forward. Something to consider doing here as well, but not blocking.

https://github.com/codeforamerica/vita-min/pull/4469/files#diff-a989d8d483d2f5596544f4765e35d8cd663fc6710402932443c8a3737565ce14R10